contracts/Core/StakeManager.sol

```diff
@@ -325,21 +383,16 @@ contract StakeManager is Utils, ACL, StakeStorage {
 
         uint256[] memory lowerCutoffsLastEpoch = _block.lowerCutoffs;
         uint256[] memory higherCutoffsLastEpoch = _block.higherCutoffs;
-        uint256[] memory mediansLastEpoch = _block.medians;
 
         if (lowerCutoffsLastEpoch.length > 0) {
             uint256 penalty = 0;
             for (uint256 i = 0; i < lowerCutoffsLastEpoch.length; i++) {
                 uint256 voteLastEpoch = voteManager.getVote(epochLastRevealed, thisStaker.id, i).value;
                 uint256 lowerCutoffLastEpoch = lowerCutoffsLastEpoch[i];
                 uint256 higherCutoffLastEpoch = higherCutoffsLastEpoch[i];
-                uint256 medianLastEpoch = mediansLastEpoch[i];
 
                 if (((voteLastEpoch < lowerCutoffLastEpoch) ||
-                    (voteLastEpoch > higherCutoffLastEpoch))) {//} &&
-                    // (voteLastEpoch != medianLastEpoch)) {
-                    //WARNING: Potential security vulnerability. Could increase stake maliciously
```

Comment:
Do not remove such critical warnings!

Reply:
WARNING: Potential security vulnerability. Could increase stake maliciously

I removed this comment because //WARNING: unchecked underflow is commented below this one, so since we have upgraded `solc` to v0.8.0 arithmetic underflow/overflow is taken care of.

This comment can be restored, but we should create a task to check whether there's vulnerability or not, if there is we should fix it. @hrishikeshio